### PR TITLE
Emissions schedule for first BIP-87 bribe payout.

### PR DESCRIPTION
**Pull request by Tritium-VLK**
For #281 :

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Nounce: Techops:237
First run results:
`Transaction was Mined 
---------------------
Tx Hash: 0x5590c6f1869d5f319d59ad69816bc87b66b7d090348acf7ffb9914cc58d588ce
From: 0x10C7a2ca16116E5C998Bfa3BC9CEF464475B18ff
To: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
Value: 0
Function: Gnosis Safe.execTransaction
Block: 14444624
Gas Used: 456191 / 513312 (88.9%)

Events In This Transaction
--------------------------
├── AdminUpgradeabilityProxy (0x0A4F4e92C3334821EbB523324D09E321a6B0d8ec)
│   ├── UnlockScheduleSet
│   │   ├── beneficiary: 0xfd05D3C7fe2924020620A8bE4961bBaA747e6305
│   │   ├── token: 0xfd05D3C7fe2924020620A8bE4961bBaA747e6305
│   │   ├── totalAmount: 22435841532164025814838
│   │   ├── start: 1648166400
│   │   ├── end: 1649376000
│   │   ├── duration: 1209600
│   │   ├── timestamp: 14444624
│   │   └── blockNumber: 1648065931
│   ├── UnlockScheduleSet
│   │   ├── beneficiary: 0x3472A5A71965499acd81997a54BBA8D852C6E53d
│   │   ├── token: 0xfd05D3C7fe2924020620A8bE4961bBaA747e6305
│   │   ├── totalAmount: 19260791644936432779264
│   │   ├── start: 1648166400
│   │   ├── end: 1649376000
│   │   ├── duration: 1209600
│   │   ├── timestamp: 14444624
│   │   └── blockNumber: 1648065931
│   └── UnlockScheduleSet
│       ├── beneficiary: 0x3472A5A71965499acd81997a54BBA8D852C6E53d
│       ├── token: 0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551
│       ├── totalAmount: 4285408409998078246912
│       ├── start: 1648166400
│       ├── end: 1649376000
│       ├── duration: 1209600
│       ├── timestamp: 14444624
│       └── blockNumber: 1648065931
│   
└── Gnosis Safe (0x86cbD0ce0c087b482782c181dA8d191De18C8275)
    └── ExecutionSuccess
        ├── txHash: 0xa93abab8e1ae6815f251c3dd3c0a55349ed828a15055217b9a4c4a1af19c9a7f
        └── payment: 0

{
│   'ethereum_client': <gnosis.eth.ethereum_client.EthereumClient object at 0x112d0e340>,
│   'safe_address': '0x86cbD0ce0c087b482782c181dA8d191De18C8275',
│   'to': '0x40A2aCCbd92BCA938b02010E17A5b8929b49130D',
│   'value': 0,
│   'data': HexBytes('0x8d80ff0a0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000034b000a4f4e92c3334821ebb523324d09e321a6b0d8ec000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4768f304f000000000000000000000000fd05d3c7fe2924020620a8be4961bbaa747e6305000000000000000000000000fd05d3c7fe2924020620a8be4961bbaa747e63050000000000000000000000000000000000000000000004c03fd91b51f87f0f3600000000000000000000000000000000000000000000000000000000623d060000000000000000000000000000000000000000000000000000000000624f7b000000000000000000000000000000000000000000000000000000000000127500000a4f4e92c3334821ebb523324d09e321a6b0d8ec000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4768f304f0000000000000000000000003472a5a71965499acd81997a54bba8d852c6e53d000000000000000000000000fd05d3c7fe2924020620a8be4961bbaa747e6305000000000000000000000000000000000000000000000414212df1059fc0000000000000000000000000000000000000000000000000000000000000623d060000000000000000000000000000000000000000000000000000000000624f7b000000000000000000000000000000000000000000000000000000000000127500000a4f4e92c3334821ebb523324d09e321a6b0d8ec000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c4768f304f0000000000000000000000003472a5a71965499acd81997a54bba8d852c6e53d000000000000000000000000937b8e917d0f36edebba8e459c5fb16f3b3155510000000000000000000000000000000000000000000000e84ffd13d38f00000000000000000000000000000000000000000000000000000000000000623d060000000000000000000000000000000000000000000000000000000000624f7b000000000000000000000000000000000000000000000000000000000000127500000000000000000000000000000000000000000000'),
│   'operation': 1,
│   'safe_tx_gas': 730898,
│   'base_gas': 0,
│   'gas_price': 0,
│   'gas_token': '0x0000000000000000000000000000000000000000',
│   'refund_receiver': '0x0000000000000000000000000000000000000000',
│   'signatures': b'',
│   '_safe_nonce': 238,
│   '_safe_version': '1.1.1',
│   '_chain_id': None
}
`

**Review by petrovska-petro — APPROVED** [review]

File: scripts/emissions/bribe_emissions_23_03_2022.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from brownie import Wei
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main(
+    total_bvecvx_emit=22435841532164025814838,
```
Comment (jorijnsmit) [inline scripts/emissions/bribe_emissions_23_03_2022.py:8]: slight diff here with actual executed tx: https://etherscan.io/tx/0x558158e201470235a1f42ab22370582ec78b9b54a9cdb99e583cd130b4f9dab9#eventlog

Comment (petrovska-petro) [reply scripts/emissions/bribe_emissions_23_03_2022.py:8]: good catch, will push commit with `Decimal` as discussed 🚀

**Comment by jorijnsmit** [comment]
closing since this got execd already: https://etherscan.io/tx/0x558158e201470235a1f42ab22370582ec78b9b54a9cdb99e583cd130b4f9dab9#eventlog

added a generalised reusable version of this script in https://github.com/Badger-Finance/badger-multisig/pull/298

**Pull request closed without merging**